docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -874,6 +874,13 @@ spring:
 ----
 
 In either case, the HTTP status of the response will be set to 401.
+Example configuration to produce http header:
+----
+# by default originalStatusHeaderName = original-http-status
+spring.cloud.gateway.setStatus.originalStatusHeaderName=my-backward-compatible-name
```

Comment:
Please use the canonical names in docs: `spring.cloud.gateway.set-status.original-status-header-name`

Reply:
done